datalab/datalab_session/data_operations/median.py

```diff
@@ -31,4 +37,50 @@ def wizard_description():
         }
     
     def operate(self):
-        pass
+        input_files = self.input_data.get('input_files', [])
+        completion_total = len(input_files)
+        image_data_list = []
+
+        log.info(f'Executing median operation on {completion_total} files')
+
+        # fetch fits for all input data
+        for index, file_info in enumerate(input_files):
+            basename = file_info.get('basename', 'No basename found')
+            
+            fits_file = find_fits(basename)
+            fits_url = fits_file[0].get('url', 'No URL found')
+
+            with fits.open(fits_url, use_fsspec=True) as hdu_list:
+                data = hdu_list['SCI'].data
```

Comment:
are these Wayne's images that have a "SCI" hdu? I'm not sure all LCO data will have that, but I guess that can be a problem for another day...

Reply:
I think we're using LCO data so they should have that? Also apparently that's the standard hdu key for fits files to store their images. But my understanding on that is shaky. It is also reffered to as hdu_list[1] aka the second hdu is typically the image data. But I did SCI since thats the standard key in case someone doesn't place it in position 1